tox.ini

```diff
@@ -1,12 +1,17 @@
 [tox]
 envlist =
     {py27,py36}-django111
+    {py36}-django{200, 210}
     flake8
     docs
 
 [testenv]
 deps =
     django111: Django>=1.11,<2.0
+    django200: Django>=2.0,<2.1
+    django210: Django>=2.1
+    django{111,200,210},docs: git+https://github.com/release-engineering/kobo.git#egg=kobo
```

Comment:
I see your patch for kobo. Let's wait a while for my question about the release date of kobo. Then, we can determine if we use this workaround or add kobo to `install_requires`.